app/domain/streams/messages/redirect_message.rb

```diff
@@ -0,0 +1,62 @@
+module Streams
+  class Messages::RedirectMessage < Messages::BaseMessage
+    def initialize(payload)
+      super
+    end
+
+    def build_attributes(base_path:, title:, warehouse_item_id:)
```

Comment:
Yeah I had realised only yesterday that I'd duplicated too much. There's enough of a difference I feel to override it here. Otherwise it'd be a lot of set to nils.